app/options.go

```diff
@@ -100,7 +100,6 @@ func ServerConnector(s *Server) AppOption {
 		a.log = s.Log
 		a.notificationsLog = s.NotificationsLog
 
-		a.accountMigration = s.AccountMigration
 		a.cluster = s.Cluster
```

Comment:
Not for this PR, but would be great to get rid of all this when is possible.

Reply:
https://mattermost.atlassian.net/browse/MM-27169